Userland/Libraries/LibWeb/HTML/HTMLCanvasElement.cpp

```diff
@@ -223,15 +223,15 @@ struct SerializeBitmapResult {
 };
 
 // https://html.spec.whatwg.org/multipage/canvas.html#a-serialisation-of-the-bitmap-as-a-file
-static ErrorOr<SerializeBitmapResult> serialize_bitmap(Gfx::Bitmap const& bitmap, StringView type, Optional<double> quality)
+static ErrorOr<SerializeBitmapResult> serialize_bitmap(Gfx::Bitmap const& bitmap, Optional<String> const& type, Optional<double> quality)
```

Comment:
Ah! Yup, I think I got a little confused here. I've left a FIXME for this, I'll try investigate this later :^)

Reply:
> Oh, our IDL generates optional DOMString type = "image/png" as an Optional<String>, doesn't it? Our generator could be smarter here, because it will never have a nullopt, but oh well.

Aha! I managed to get this working :^) I'll try fix it up here so we won't have to fix this up later.